Unity3D/Assets/RosSharp/Editor/UrdfImporter/UrdfRobotExtensions.cs

```diff
@@ -59,5 +78,93 @@ public static void setKinematic(GameObject robot, bool isKinematic)
             foreach (Rigidbody rigidbody in rigidbodies)
                 rigidbody.isKinematic = isKinematic;
         }
+
```

Comment:
The method ImportUrdfToUnity() does many things:
* creating folders
* searching files
* asking for user input
* searching the xml
* copying mesh files
* saving the urdf file contents
* and likely more things which I have overlooked when writing this list.

Please split them into spearate methods.

Reply:
@MartinBischoff I've split into separate methods. Let me know what you think.